docs(kubernetes) : Adding instruction for the messenger queue with Fr… (#1939)
api-platform/docs: deployment/kubernetes.md

The `readinessProbe` and the `livenessProble` can not use the default `docker-healthcheck` but should test if the command is running.

> If you are using the FrankenPHP image make sure to install the `/bin/ps` binary (just add `procps` in the `apt-get install script`), otherwise the `readinessProbe` and `livenessProbe` will fail.

Reviewer: This will be necessary as long as Debian (even without FrankenPHP). So I suggest the following:

```suggestion
Finally, make sure to install the `/bin/ps` binary), otherwise the `readinessProbe` and `livenessProbe` will fail:

```patch
# api/Dockerfile

RUN apt-get update && apt-get install --no-install-recommends -y \
+	procps \
```
```
